extensions/amp-animation/0.1/storybook/Basic.amp.js

```diff
@@ -50,30 +49,38 @@ const BLOCK_STYLE = {
   height: '100px',
 };
 
-// eslint-disable-next-line
-storiesOf('Animation', module)
-  .addDecorator(withKnobs)
-  .addDecorator(withAmp)
-  .addParameters({extensions: [{name: 'amp-animation', version: 0.1}]})
-  .add('default', () => {
-    const keyframesOptions = Object.keys(KEYFRAMES_OPTIONS);
-    const keyframesName = select(
-      'Keyframes',
-      keyframesOptions,
-      keyframesOptions[0]
-    );
-    const keyframes = KEYFRAMES_OPTIONS[keyframesName];
-    const easing = text('Easing', 'cubic-bezier(0,0,.21,1)');
-    const spec = {
-      animations: {
-        selector: '#block',
-        easing,
-        keyframes,
-      },
-    };
-    return (
-      <AnimationTemplate spec={spec}>
-        <div id="block" style={BLOCK_STYLE} />
-      </AnimationTemplate>
-    );
-  });
+export default {
+  title: 'Animation',
+  decorators: [withKnobs, withAmp],
+
+  parameters: {
+    extensions: [{name: 'amp-animation', version: 0.1}],
+  },
+};
+
+export const Default = () => {
```

Comment:
Could you pls check if the `_default` name is recommended in these cases?

Reply:
(As in other thread)

Could not find `_default` in their docs, which would contradict this:

> [**Named story exports**](https://storybook.js.org/docs/react/api/csf#named-story-exports)
> ...
> It's recommended to start export names with a capital letter.

But if we prefer `_default` I'd be happy to make the change. We can consider `_Default` too.

Reply:
Oh, and this is how the provided codemod modified that name automatically.

Reply:
Let's keep as you have then.